### PR TITLE
Switch from setupController to a simpler model hook

### PR DESCRIPTION
**Pull request by michaelrkn**
Keeps things more straightforward so that the reader can focus on the issue at hand, rather than getting potentially confused by a less-common way of setting the model.

### CODE REVIEW

File: source/controllers/representing-multiple-models.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-playlist:
+one. Here, our router's `model` hook returns an array of songs:
 
 ```app/routes/songs.js
 export default Ember.Route.extend({
-  setupController(controller, playlist) {
-    controller.set('model', playlist.get('songs'));
-  }
+ model: function() {
+   this.store.findAll('song');
```
Comment (irnc) [inline source/controllers/representing-multiple-models.md]: This should be `return this.store.findAll('song');`

**Comment by michaelrkn** [comment]
Fixed! Thanks for the heads up, @irnc!

File: source/controllers/representing-multiple-models.md [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
-In our route, we setup our `SongsController` to represent the songs in a
-playlist:
+one. Here, our router's `model` hook returns an array of songs:
 
 ```app/routes/songs.js
 export default Ember.Route.extend({
-  setupController(controller, playlist) {
-    controller.set('model', playlist.get('songs'));
-  }
+ model: function() {
```
Comment (knownasilya) [inline source/controllers/representing-multiple-models.md]: Probably keep the shorthand version, i.e. `model() {`

**Comment by michaelrkn** [comment]
does this look right @knownasilya?

**Comment by knownasilya** [comment]
:+1:

**Comment by michaelrkn** [comment]
Thanks for the help! @rwjblue can you merge this?

File: source/controllers/representing-multiple-models.md [hunk 3]
```diff
@@ -1,14 +1,10 @@
 The model of a controller can represent several records as well as a single
-one. Here we will customize the default routing logic and set the `model`
-property in a route's `setupController` method.
-
-In our route, we setup our `SongsController` to represent the songs in a
-playlist:
+one. Here, our router's `model` hook returns an array of songs:
```
Comment (HeroicEric) [inline source/controllers/representing-multiple-models.md]: `router's` -> `route's`?

Comment (michaelrkn) [inline source/controllers/representing-multiple-models.md]: good catch! thank you. fixed

**Pull request merged**